src/Stryker.Core/Stryker.Core/TestRunners/DotnetTestRunner.cs

```diff
@@ -18,15 +18,16 @@ public DotnetTestRunner(string path, IProcessExecutor processProxy)
 
         public DotnetTestRunner(string path) : this(path, new ProcessExecutor()) { }
 
-        public TestRunResult RunAll()
+        public TestRunResult RunAll(int timeoutMS = 0)
```

Comment:
I think a `int? timeoutMS = null` is clearer than using `0` as a magical number that means `off`